src/flick/like/tests/test_like_and_unlike.py

```diff
@@ -74,3 +75,33 @@ def test_like_and_cancel(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(data["num_likes"], 0)
         self.assertFalse(data["has_liked"])
+
+    def test_lst_like_and_cancel(self):
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.friend_token)
+
+        # test like
+        response = self.client.post(self.LIKE_LST_URL)
+        data = json.loads(response.content)["data"]
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(data["num_likes"], 1)
+
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.user_token)
+
+        # test notification is received
+        response = self.client.get(self.NOTIFICATION_URL)
+        data = json.loads(response.content)["data"]
+        self.assertEqual(response.status_code, 200)
```

Comment:
assert status code before you parse data

Reply:
fixed :p